neutrinordp/xrdp-neutrinordp.c

```diff
@@ -94,10 +94,107 @@ lxrdp_start(struct mod *mod, int w, int h, int bpp)
 
 /******************************************************************************/
 /* return error */
+static void
+set_keyboard_overrides(struct mod *mod)
+{
+    const struct kbd_overrides *ko = &mod->kbd_overrides;
+    rdpSettings *settings  = settings = mod->inst->settings;
```

Comment:
You'll have seen that the CI code check is complaining about this line. It should be :-

```
rdpSettings *settings  = mod->inst->settings;
```

Looking back at the conversation, I can see that I suggested it in the first place. Apologies.

Reply:
It's not your fault.
I had pasted too easily.
I should have considered it carefully.

I will correct it.